plugins/PromisedScene/main.js

```diff
@@ -320,40 +339,62 @@ module.exports = async ({
       }
     }
 
-    result.description = await questionAsync(
-      "What is the DESCRIPTION for the scene?: ",
+    const manualEnterDescriptionOfScene = await questionAsync(
+      {
+        type: "input",
+        name: "DescriptionOfScene",
+        message: "What is the DESCRIPTION for the scene?: ",
+      },
       "TESTMODE Question MANUAL Description",
       testMode && testMode.questions ? testMode.questions.manualDescription : ""
     );
 
+    result.description = manualEnterDescriptionOfScene.DescriptionOfScene;
+
     const splitactors = await questionAsync(
-      `What are the Actors NAMES in the scene?: (seperated by Comma) ${
-        actor.length ? ` ${actor.join(", ")}` : ""
-      }`,
+      {
+        type: "input",
+        name: "ActorsOfScene",
+        message: `What are the Actors NAMES in the scene?: (seperated by Comma)`,
+        default() {
+          return `${actor.length ? ` ${actor.join(", ")}` : ""}`;
+        },
+      },
       "TESTMODE Question MANUAL actor names",
       testMode && testMode.questions ? testMode.questions.manualActors : ""
     );
 
-    const areActorsBlank = splitactors === "" || splitactors === " " || splitactors === null;
+    const areActorsBlank =
+      splitactors.ActorsOfScene === "" ||
+      splitactors.ActorsOfScene === " " ||
+      splitactors.ActorsOfScene === null;
```

Comment:
Simpler, and handles a string that has more than one space. Can copy to other "are blank" variables

```suggestion
    const areActorsBlank = !splitactors.ActorsOfScene || !splitactors.ActorsOfScene.trim();
```